app/src/hooks/api/useSurveyApi.ts

```diff
@@ -133,6 +134,18 @@ const useSurveyApi = (axios: AxiosInstance) => {
     return data;
   };
 
+  /**
+   * Get permit numbers applicable to use for a survey based on project ID
```

Comment:
Rather than 'applicable to use' maybe say something like: "Get permit numbers that have not already been assigned to a survey, by project ID."

Or alternatively, add a note: "Note: applicable permits are those not already assigned to a survey"

Or some combination of both